docs/src/components/App/Header/Header.js

```diff
@@ -20,13 +20,14 @@ import { generateRouteList } from './headerUtils';
 import demoSpecExports from '../../../../../react/*/*.demo.js';
 const demoSpecs = demoSpecExports.map(x => x.default);
 const allRoutes = [
+  { route: '/palette', title: 'Colour Palette', category: 'Guides' },
   { route: '/typography', title: 'Typography', category: 'Guides' },
   { route: '/page-layout', title: 'Page Layout', category: 'Guides' },
   {
     route:
-      process.env.NODE_ENV === 'production' ?
-        `${process.env.BASE_HREF}playroom` :
-        'http://localhost:9000/',
+      process.env.NODE_ENV === 'production'
+        ? `${process.env.BASE_HREF}playroom`
+        : 'http://localhost:9000/',
```

Comment:
Looks like a fight with prettier, maybe revert the formatting changes, and we should prettier-ify this repo in another PR as eslint isn't happy

Reply:
Going to wait for @jahredhope 's PR to add prettier to SSG.